src/components/Header/AppBanner.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {Container, Icon} from 'semantic-ui-react';
+import {useQuery} from '@apollo/react-hooks';
+import {ALL_BANNERS} from '../../admin/queries';
+
+const AppBanner = () => {
+  const {loading, error, data} = useQuery(ALL_BANNERS);
+  const iconMap = {
+    INFO: 'info circle',
+    WARN: 'warning circle',
+    ERROR: 'times circle',
+  };
+
+  const isActive = (banner) => {
+    /* 
+    Banner is active if it is enabled && 
+    start date <= today && end date is after today 
+    */
+    if (!banner.startDate || !banner.endDate) {
+      return false;
+    }
+    const today = new Date().setHours(0, 0, 0, 0);
+    return (
+      banner.enabled &&
+      Date.parse(banner.startDate) <= today &&
+      Date.parse(banner.endDate) > today
+    );
+  };
+
+  if ((!loading || !error) && data && data.allBanners.edges.length > 0) {
+    console.log('How many banners to evaluate');
+    console.log(data.allBanners.edges);
+    const banners = data.allBanners.edges
+      .map((banner) => banner.node)
+      .filter((banner) => isActive(banner));
+    const displayBanners = banners
+      .slice()
+      .sort((a, b) => a.startDate - b.startDate);
+
+    console.log('**** Displaying banners');
+    console.log(banners);
+
+    return (
+      displayBanners.length > 0 && (
+        <Container fluid className="app-banner" textAlign="center">
```

Comment:
This container is very large and takes up a lot of real-estate on every page. Maybe scale it down to about half the size?

Reply:
Ok, let me try moving it below the main nav bar and scaling it down. Do you think it should take up the full width of the page or no?

Reply:
Full page width is good so long as the content is within the container.